mmv1/products/tags/api.yaml

```diff
@@ -0,0 +1,99 @@
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+objects:
+  - !ruby/object:Api::Resource
+    name: 'TagKey'
+    min_version: beta
+    base_url: tagKeys
+    self_link: "{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    description: A TagKey, used to group a set of TagValues.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/resource-manager/docs/tags/tags-creating-and-managing'
+      api: 'https://cloud.google.com/resource-manager/reference/rest/v3/tagKeys'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          Immutable. The resource name for a TagKey. Must be in the format tagKeys/{tag_key_id}, where tag_key_id is the generated numeric id for the TagKey.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: parent
+        description: |
+          Immutable. The resource name of the new TagKey's parent. Must be of the form organizations/{org_id}.
+        input: true
+        required: true
+      - !ruby/object:Api::Type::String
+        name: shortName
+        description: |
+          Required. Immutable. The user friendly name for a TagKey. The short name should be unique for TagKeys within the same tag namespace.
+
+          The short name must be 1-63 characters, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
+        input: true
+        required: true
+      - !ruby/object:Api::Type::String
+        name: namespacedName
+        description: |
+          Output only. Immutable. Namespaced name of the TagKey.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: description
+        description: |
+          Optional. User-assigned description of the TagKey. Must not exceed 256 characters.
+      - !ruby/object:Api::Type::String
+        name: createTime
+        description: |
+          Output only. Creation time.
+
+          A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+        output: true
+      - !ruby/object:Api::Type::String
+        name: updateTime
+        description: |
+          Output only. Update time.
+
+          A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+        output: true
+      - !ruby/object:Api::Type::String
+        name: etag
```

Comment:
I don't think we want to expose the `etag` field to users. Generally this is used to prevent race conditions, but we should either handle it internally within Terraform or not use it at all.

IAM policies also have etags, and handle them internally within the provider

Reply:
👍  I'll remove it for now.